src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapConnection.cs

```diff
@@ -1118,19 +1105,19 @@ private void BindHelper(NetworkCredential newCredential, bool needSetCredential)
                 }
 
                 tempDomainName.Append(username);
-                error = Wldap32.ldap_simple_bind_s(_ldapHandle, tempDomainName.ToString(), password);
+                error = NativePal.LdapSimpleBind(_ldapHandle, tempDomainName.ToString(), password);
```

Comment:
The other half of NativePal would be LdapPal:

```suggestion
                error = LdapPal.SimpleBind(_ldapHandle, tempDomainName.ToString(), password);
```

Reply:
I am planning on separating the Interop files as you suggested in a different comment to have Interop.Ldap.cs and Interop.Ber.cs (under Common folder) but is it also necessary to separate the Pals? Given that the Pal's purpose is to redirect to the right library, and that both ldap and ber calls go to the same native library in both platforms, i would have thought that this wouldn't be required.